app/views/team_users/create.js.erb

```diff
@@ -3,9 +3,9 @@
 <% else %>
   $("<%= escape_javascript(render @team_user) %>").appendTo("#team_users");
   <% if @promoted_role %>
-    $('#float-alert p').html("New user added to the team (promoted to owner because it is a Portus admin).");
+    $('#float-alert p').html("User '<%= escape_javascript(User.find(@team_user.user_id).username) %>' was added to the team (promoted to owner because it is a Portus admin).");
```

Comment:
Instead of `User.find(@team_user.user_id).username)`, you can do `@team_user.user.username)`